docs/what-is-zarf.md

```diff
@@ -0,0 +1,225 @@
+
+As well as a lightweight, long-running k8s cluster&mdash;called "**the Zarf cluster**"&mdash;which:
+
+- hosts the services necessary for exposing package contents to your isolated, downstream consumers.
+
+&nbsp;
+
+
+## How it's used (from 30,000 ft)
+
+Zarf is intended for use in a software deployment process that looks something like this:
+
+<a href="../.images/what-is-zarf/how-to-use-it.png">
+  <img alt="how it works" src="../.images/what-is-zarf/how-to-use-it.png" heigth="262" />
+</a>
+
+### (0) - The internet
+
+Zarf wants to go out into the world to pull & package application software, so 1) the world has to exist, and 2) you need to make sure Zarf can reach the required resources within it (i.e. has a connection to the internet).
+
+That should probably go without saying, but... it's always the "little things" that trip you up, right?
+
+### (1) - Zarf builds you a package
+
+You feed the `zarf` binary a "recipe" (`zarf.yaml`) and it makes itself busy downloading, packing, and compressing the software you asked for. It outputs a single, ready-to-move distributable (cleverly) called "a package".
+
+Find out more about what that looks like in the "[Building a package](#building-a-package)" section.
+
+### (2) - You ship the package
+
+When it comes to remote / constrained / independent / air gapped systems, everyone's unique. Zarf doesn't have an opinion as to _how_ packages move just so long as you can get them into your downstream environment.
+
+Find out more about what that _might_ look like in the "[Shipping a package](#shipping-a-package)" section.
+
+### (3) - Zarf publishes your package
+
+Once your package has landed, depending on what you've built into it, there are a couple of ways for Zarf to expose package contents.
+
+In the first scenario, your package consists of a single application (plus dependencies) and you configure the Zarf cluster to _serve your application directly to end users_. This mode of operation is called "Appliance Mode"&mdash; because it's small & self-contained like a kitchen appliance&mdash;and it is intended for use in environments where you want to run k8s-native tooling but need to keep a small footprint (i.e. single-purpose / constrained / "edge" environments).
+
+Find out more about what direct user service looks like in the "[Appliance Mode](#appliance-mode)" section.
+
+In the second scenario, your package consists of updates for many apps / systems and you configure the Zarf cluster to _propagate updates to downstream systems_ rather than to serve users directly. This mode of operation is called "Utility Cluster Mode"&mdash;as it's main job is to _add utility_ to other clusters&mdash;and it is intended for use in places where you want to run independent, full-service production environments (your own [Big Bang](https://github.com/DoD-Platform-One/big-bang) cluster, perhaps?) but you need help tracking, caching & disseminating system / dependency updates.
```

Comment:
Having the words `configure the Zarf cluster` and `is called "Utility Cluster Mode"` somewhat clash for me since it wouldn't be unreasonable for a new Zarfer to think there are now two clusters that just got spun up. 

What do you think of making it read more like `In the second scenario, called "Utility Cluster Mode", ........... you configure the Utility cluster to ....`?

Reply:
Is `Zarf cluster` a term we want to be a thing?

Reply:
Mmm, I think it's useful to have "Zarf cluster" be a thing, yes.  Because, in either scenario, there _is_ always only one cluster that Zarf touches--the "Zarf(-controlled) cluster".

It's a matter of how we stuck another label on top of it depending on how it's being used: in "applicance" vs "utility cluster" mode.  I was trying to mirror the terminology I've heard in conversation but perhaps that's not helping..?

What about if:

> Once your package has landed, depending on what you've built into it, there are a couple of ways for Zarf to expose package contents.

became,

> Once your package has landed, depending on what you've built into it, there are a couple of ways for **the Zarf cluster** to expose package contents.

Does that make it any clearer that there is _a single Zarf cluster_ (independent of usage) and that the "appliance" vs "utility" terms are monikers for the usage scenario rather than the cluster itself?

Reply:
I'd be happy to come up with some totally separate terms for the "modes" if that helps, too.  Or, maybe just dropping the word "cluster" from the "modes" altogether--as in: "Appliance Mode" and "Utility Mode".  That might make it more apparent that we expect there to always be "the Zarf cluster" and it can be used in a couple of different ways.

I've seen quite a bit of confusion from the people we've been demo'ing to thinking that Zarf was going to (somehow) stand up their entire production cluster _for them_--driven by how we're doing "appliance mode" demos, probably--whereas Zarf was always conceived of (I think?) as a "cluster to sit next to" the production cluster.

Reply:
> it wouldn't be unreasonable for a new Zarfer to think there are now two clusters that just got spun up

Interesting. The "Zarf publishes your package" block reads like something you'd do _all_ of at the same time?  I tried to use the "scenarios" to make it clear that this was an "either-or" thing (for the purposes of this write-up, anyway) but that didn't come across, eh?

What if we make the lead-ins more like:
> In the first, simple scenario, ...

and...
> In a totally separate, more intensive scenario, ...

Does that make it clearer that these should be considered exclusively?